libs/tree-sitter-wing/src/scanner.c

```diff
@@ -150,6 +151,42 @@ static bool scan_automatic_semicolon(TSLexer *lexer)
   return true;
 }
 
+/**
+ * Check if an automatic empty block ( `{}` ) could be inserted
+ *
+ * @return true if an automatic block was found (aka "inserted"), false otherwise
+ */
+static bool scan_automatic_block(TSLexer *lexer)
+{
```

Comment:
I have no idea how the tree sitter parser works but it looks dope. Would be cool to learn more about it in a team time or something 😄